infraohjelmointi_api/migrations/0046_coordinatornote.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.1.3 on 2023-09-28 07:21
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('infraohjelmointi_api', '0045_auto_project_finances_to_frame_finances'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CoordinatorNote',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('coordinatorNote', models.TextField(blank=True, default='')),
+                ('year', models.CharField(max_length=4)),
+                ('planningClass', models.CharField(max_length=150)),
+                ('updatedByFirstName', models.CharField(max_length=50)),
+                ('updatedByLastName', models.CharField(max_length=50)),
+                ('createdDate', models.DateTimeField(auto_now_add=True)),
+                ('planningClassId', models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='infraohjelmointi_api.projectclass')),
```

Comment:
Notes are expected to be visible on coordination classes, would it be more appropriate for the field to be called coordinatorClass?

Validation can also be added to check if the class assigned to this field is a coordinator class. `ClassFinancial.py` model and `ClassFinancialSerializer.py `can be checked for example.